tfhe/src/high_level_api/details.rs

```diff
@@ -1,4 +1,4 @@
-#[cfg(any(feature = "boolean", feature = "shortint", feature = "integer"))]
```

Comment:
Why remove integer here?

Reply:
This macro is (and was) not used in the integer part of the hlapi, so it should not have been there